### ISSUE TITLE
[Feature]: Improve the documentation

### ISSUE THREAD

**Issue by Youssef535** [react-icons/react-icons#703]
### Describe

i'm always frustrated when i use the documentation of react icons the team need to Improve the docs and wrok more on it like other lib.

### Solution

The best solution is the rewrite the docs again and improve with How to and provide more information in it.

**Comment by ChetSocio** [comment]
This project has very nice documentation for real!  Its so easy to set up and run that you find it irritating? Just `npm install react-icons --save`  and search for icons that you like using ctrl+f . What more you need? Icons are classsified properly too.You just wastin the time

**Comment by Youssef535** [comment]
What about styling and Accessibility please look at other docs like icons bootstrap...

**Comment by ChetSocio** [comment]
I usually use styling like `<MyIcon className="" /> `and it works. I think this package is collection of many icons packages  and you have to explore them to customize the icons. Usually I don't need these as I have no problems with normal react-icons at all.

**Comment by Youssef535** [comment]
Yes but we can also use <FaBeer  style = 5px/> like an example if you keep using vanilla css into all your components it will affect your app sooner or later (performance).
